src/webprojects/sumo.md

```diff
@@ -61,8 +61,11 @@ Only when there is a release or a major change will there be communications to l
 
 ### What are the ways to report an issue?
 
-The best place to report issues is by filing [bugs to SUMO](https://bugzilla.mozilla.org/enter_bug.cgi?product=support.mozilla.org&component=Localization). This is the quickest way to have a problem resolved.
-
-Alternatively, you can report a problem through the SUMO l10n forum. You can also raise questions through [web projects mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web), in the #sumo channel on [Discourse](https://discourse.mozilla.org/c/sumo/), and other social channels. In the end, issues identified will be tracked through Bugzilla.
+* File a [bug to SUMO](https://bugzilla.mozilla.org/enter_bug.cgi?product=support.mozilla.org&component=Localization). This is the quickest way to have a problem resolved.
+* Discuss it in [the SUMO l10n forum](https://support.mozilla.org/forums/l10n-forum)
+* Raise questions in the #sumo channel on [Discourse](https://discourse.mozilla.org/c/sumo/)
+* Ping in [the Matrix channel](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org) to the l10n PM responsible for the project.
+* Comment on the string in Pontoon and ping the L10n PM responsible for the project.
```

Comment:
Maybe mention the `REQUEST CONTEXT or REPORT ISSUE` button instead, explaining that this will start a comment flagging the PM in charge.